.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Environment**
+- __Package version(s)__: <!-- fill this out -->
+- __Operating System__: <!-- fill this out -->
+- __Browser name and version__: <!-- fill this out -->
+
+**Code Sandbox**
+Link to a minimal repro (fork [this code sandbox](https://codesandbox.io/s/itwinui-react-example-son74)): <!-- here -->
+
+**Steps to reproduce**
+1. <!-- fill this out -->
+1. <!-- fill this out -->
+1. <!-- fill this out -->
+
+**Actual behavior**
+A clear and concise description of what the bug is.
```

Comment:
It is shown on the same line. Better make it to move to new line.

Reply:
fixed